aten/src/ATen/native/cuda/Activation.cu

```diff
@@ -522,6 +522,38 @@ void hardswish_backward_kernel(TensorIterator& iter) {
   });
 }
 
+void hardsigmoid_kernel(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.dtype(), "hardsigmoid_cuda", [&]() {
```

Comment:
Tensorflow uses a different definition, https://www.tensorflow.org/api_docs/python/tf/keras/activations/hard_sigmoid,
we do not have a use case for that definition, so lets stick with this one